src/runtime_src/core/edge/include/xclhal2_mpsoc.h

```diff
@@ -27,7 +27,7 @@ extern "C" {
 #define	XRT_MAX_NAME_LENGTH	32
 #define	XRT_MAX_PATH_LENGTH	255
 
-#define	SOFT_KERNEL_FILE_PATH	"/home/softkernel/softkernel/"
+#define	SOFT_KERNEL_FILE_PATH	"/home/petalinux/softkernel/"
```

Comment:
Just try to understand why we can not use 'softkernel' as is the case on U30?

Reply:
Currently there is only petalinux user on petalinux build
If there is a way to create softkernel user then I can switch back to use softkernel

Reply:
> Currently there is only petalinux user on petalinux build If there is a way to create softkernel user then I can switch back to use softkernel

Does the platform customize the petalinux? If so, please refer to https://jira.xilinx.com/browse/CR-1085088

Reply:
I am not aware of any customization for petalinux.
However, I found a way to add softkernel user to apu-boot script which can create softkernel user.